### PR TITLE
Create the AI features page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Built the new AI features page.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/ai/features](http://0.0.0.0:8001/ai/features)
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1DT5nCRSJH-OkvDmlyKh6vVhofeHPHZJHTTBDNu44wCY/edit#)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3341

### CODE REVIEW

File: templates/ai/features.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+{% block content %}
+
+<section class="p-strip is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1>Canonical&rsquo;s AI and ML features</h1>
+      <p class="p-heading--three">Architectural freedom. Fully automated operations. Accelerated Deep Learning</p>
+      <p>Canonical&rsquo;s Kubeflow on Ubuntu gives you the flexibility to place your AI, ML and DL services exactly where you want them while sharing operational code with a large community.</p>
+      <p><a href="/ai/contact-us?product=ai-features">Contact us about Kubeflow&nbsp;&rsaquo;</a></p>
```
Comment (steverydz) [inline templates/ai/features.html]: This is a button in the copy doc

File: templates/ai/features.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    </div>
+    <div class="p-card col-6">
+      <div class="p-card__header">
+        <h3>Just learning AI? Start with Kaggle</h3>
+      </div>
+      <p>Kaggle competitions are a great way to start learning about AI and develop your skills. For beginners, consider starting with one of the following previous competitions:</p>
+      <ul class="p-list--divided">
+        <li class="p-list__item"><a href="https://www.kaggle.com/c/titanic" class="p-link--external">Binary classification with &lsquo;Titanic: Machine Learning from Disaster&rsquo;</a></li>
+        <li class="p-list__item"><a href="https://www.kaggle.com/c/forest-cover-type-prediction" class="p-link--external">Multi-class classification with &lsquo;Forest Cover Type Prediction&rsquo;</a></li>
+        <li class="p-list__item"><a href="https://www.kaggle.com/c/bike-sharing-demand" class="p-link--external">Binary classification with &lsquo;Titanic: Machine Learning from Disaster&rsquo;</a></li>
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/ai/features.html]: Done

File: templates/ai/features.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  <div class="row u-equal-height">
+    <div class="p-card col-6">
+      <div class="p-card__header">
+        <h3>Effective decision making</h3>
+      </div>
+      <p>With deep learning on vast amounts of data, make quicker and more effective decisions. Over time, the machine algorithms learn to distinguish what data is important what isn&rsquo;t. Insight extracted from AI will allow you to optimize your processes.</p>
+    </div>
+    <div class="p-card col-6">
+      <div class="p-card__header">
+        <h3>Improve SLA with operational predictions</h3>
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/ai/features.html]: Done

File: templates/ai/features.html [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  </div>
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-8">
+      <h2>Tensorflow&trade;</h2>
+      <p>TensorFlow is an open source software library for high performance numerical computation. Its flexible architecture allows easy deployment of computation across a variety of platforms (CPUs, GPUs, TPUs), and from desktops to clusters of servers to mobile and edge devices. Originally developed by researchers and engineers from the Google Brain team within Google&rsquo;s AI organization, it comes with strong support for machine learning and deep learning and the flexible numerical computation core is used across many other scientific domains.</p>
+      <p><a href="https://www.tensorflow.org/" class="p-link--external">Learn more about Tensorflow</a>
+    </div>
+    <div class="col-4 u-hide--small">
+      <p><img src="{{ ASSET_SERVER_URL }}710ddd41-mnist_tensorboard.png" alt="" /></p>
+      <cite>TensorFlow comes with visualization technology &mdash; TensorBoard. It features graphs, histograms, and helps with visualizing learning.</cite>
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/ai/features.html]: Done

File: templates/ai/features.html [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+  <div class="row u-equal-height">
+    <div class="p-card--highlighted col-6">
+      <div class="p-card__header">
+        <h3>Networking</h3>
+      </div>
+      <p>Network performance is critical for speeding up large deep learning exercises. The major factor in perceived cloud performance is aggregate network throughput and latency across the underlying cluster. Canonical&rsquo;s work with hyperscale public clouds ensures that we have deep insight into the dynamics of cloud network performance and security best practices for large-scale multi-tenanted operations. Our work with telco groups for NFV and edge clouds ensures that we can work well in complex environments where latency and security are critical.</p>
+    </div>
+    <div class="p-card--highlighted col-6">
+      <div class="p-card__header">
+        <h3>Operational Dashboards</h3>
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/ai/features.html]: Done

File: templates/ai/features.html (hunk 1 above)
Comment (anthonydillon) [reply templates/ai/features.html]: Done

**Comment by anthonydillon** [comment]
@pmahnke @steverydz Addressed the comments above.

**Comment by anasereijo** [comment]
@anthonydillon screencapture-ubuntu-pr-3446-run-demo-haus-ai-features-2018-06-26-14_27_23

**Comment by frankiegranato** [comment]
- [ ] Swap image and text on the JupyterHub section (to solve the embiggened image problem)
- [ ] Add footer in the conventional way (adjust dashboard image)

**Comment by anthonydillon** [comment]
@anasereijo @frankiegranato done

**Comment by anthonydillon** [comment]
@anasereijo @pmahnke done

**Comment by anasereijo** [comment]
@anthonydillon thanks for making those changes! Looks good 👍  The Jupyter image looks very low res. I checked their website and the image looks much better. Is there anything you can do about it? Let me know if you need my help. Thanks

**Pull request merged**